aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -154,6 +155,7 @@ def _init_tracing(
         span_exporter = _customize_exporter(span_exporter, resource)
         trace_provider.add_span_processor(BatchSpanProcessor(span_exporter))
 
+    _export_unsampled_span_for_lambda(trace_provider, resource)
```

Comment:
nit: can we place this logic inside `_customize_span_processors` function where we set the custom span processors for trace provider. It seems will be organized. A picky comment :-0